apps/arweave/src/ar_mining_stats.erl

```diff
@@ -334,7 +335,7 @@ get_overall_average(PartitionPeer, Stat, TotalCurrent) ->
 	end.
 
 get_partition_data_size(PartitionNumber) ->
-    Pattern = {{partition, PartitionNumber, storage_module, '_'}, '$1'},
+    Pattern = {{partition, PartitionNumber, storage_module, '_', packing, '_'}, '$1'},
```

Comment:
We sum all present packings but mine with only one key so the reported metrics are skewed. Should not we look for the mining address-related packing here?

Reply:
ah yeah, you're right.

Reply:
Updated in latest commits